### ISSUE TITLE
[Accessibility][Usability]A11y_VSCode_ViewSearch_Keyboard: Keyboard shortcuts not present for easy access of few buttons. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52607]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:** 
Issue Type : Usability

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Search something in "Search" field and navigate back to buttons - "Clear", "Refresh" and "Collapse All" button.

**Actual:**
User has to navigate using "Shift + Tab" each time he needs to perform any action on "Search" field like refresh, clear etc. 

**Expected:**
Keyboard shortcuts should be provided for the "Clear", "Refresh" and "Collapse All" buttons.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard only users will need to navigate multiple times to those buttons to perform basic functionality with those buttons.

**Attachment For Reference:**
[A11y_VSCode_ViewSearch_Keyboard_Usability_KeyboardShortcuts.pptx](https://github.com/Microsoft/vscode/files/2123681/A11y_VSCode_ViewSearch_Keyboard_Usability_KeyboardShortcuts.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
It's possible for the user to add their own keybindings. We don't provide keybindings out of the box for every action.

I just registered the 'refresh' action as a command so a user can add a keybinding, the other two are already there.
